zilliqa/src/node.rs

```diff
@@ -119,10 +119,21 @@ impl Node {
         match message {
             Message::External(external_message) => match external_message {
                 ExternalMessage::Proposal(m) => {
+
+                    let m_view = m.header.view;
+
                     if let Some((leader, vote)) = self.consensus.proposal(m)? {
                         self.reset_timeout.send(())?;
                         self.message_sender
                             .send_external_message(leader, ExternalMessage::Vote(vote))?;
+                    } else {
+                        if m_view % 5 == 0 {
```

Comment:
This is quite crude but just to show that the nodes don't need to be included immediately